src/main/java/dk/kb/datahandler/util/HarvestTimeUtil.java

```diff
@@ -39,7 +47,12 @@ public static synchronized String loadLastHarvestTime(OaiTargetDto oaiTarget) th
     
     public static synchronized void updateDatestampForOaiTarget(OaiTargetDto oaiTarget, String datestamp) throws Exception{        
         
-        if (!validateDataFormat(datestamp)) {
+        //Hack to fix datestamp return from Preservica. Format returned are not in OAI standard and is parsed wrong (downgrade to day) when given to preservica.
+        if (datestamp.length() >20) { // 2021-03-24T19:57:34.123Z -> 2021-03-24T19:57:34Z 
+            datestamp=datestamp.substring(0,19)+"Z";
+        }
```

Comment:
A benign hack, so all ok there. The comment says "downgrade to day", but the code downgrades to seconds.

Reply:
Fixed.. Comment was not updated after I gave up using from-until in yyyy-MM-dd format for preservica.